docs/pages/demo/datepicker/CustomDay.example.jsx

```diff
@@ -33,9 +33,9 @@ function WeekPicker(props) {
   const classes = useStyles(props);
   const [selectedDate, handleDateChange] = useState(new Date());
 
-  const renderWeekPickerDay = (date, selectedDate, DayComponentProps) => {
+  const renderWeekPickerDay = (date, selectedDates, DayComponentProps) => {
     let dateClone = makeJSDateObject(date);
-    let selectedDateClone = makeJSDateObject(selectedDate);
+    let selectedDateClone = makeJSDateObject(selectedDates[0]);
```

Comment:
🤦‍♂️  it is fail

You are right, we changed calendar selected days to be an array when implementing date range picker. Here will be enough to only rename argument (because we have selected date available as state variable) 

> But this makes me think that we don't need this argument at all